Userland/Libraries/LibGL/SoftwareGLContext.h

```diff
@@ -176,10 +180,7 @@ class SoftwareGLContext : public GLContext {
     FloatVector4 m_current_vertex_tex_coord = { 0.0f, 0.0f, 0.0f, 1.0f };
     FloatVector3 m_current_vertex_normal = { 0.0f, 0.0f, 1.0f };
 
-    Vector<GLVertex, 96> vertex_list;
-    Vector<GLTriangle, 32> triangle_list;
-    Vector<GLTriangle, 32> processed_triangles;
-    Vector<GLVertex> m_clipped_vertices;
+    Vector<SoftGPU::Vertex, 96> m_vertex_list;
```

Comment:
This 96 feels really arbitrary. I can't quite remember why it's exactly this size. Should we perhaps put it into a `constexpr` if it's related to the standard?

Reply:
I will remove the 96 since we always `clear_with_capacity()` this vector